### PR TITLE
[feat](HistoricalDataChart): Add Historical Rewards from New API

### PR DESCRIPTION
**Pull request by fantzulin**
## Types of changes

- **New feature**

## Description

Display historical rewards on the line chart.

## Checklist:

- [ ] Fetch data from the `/rewards/historical-data?claimableUser=${userAddress}` API.

### CODE REVIEW

File: pages/views/HistoricalDataChart.jsx [hunk 1]
```diff
@@ -8,9 +11,14 @@ const Line = dynamic(
 );
 const HistoricalDataChart = () => {
   const [data, setData] = useState([]);
-
+  const WEB3_CONTEXT = useContext(web3Context);
+  const userAddress = WEB3_CONTEXT?.address;
   useEffect(() => {
-    asyncFetch();
+    if (userAddress == "0x038919c63aff9c932c77a0c9c9d98eabc1a4dd08") {
```
Comment (fantzulin) [inline pages/views/HistoricalDataChart.jsx:17]: 目前是把錢包地址寫死，未來應該會在這邊判斷是不是付費用戶

Comment (david30907d) [reply pages/views/HistoricalDataChart.jsx:17]: 讚讚

**Review by david30907d — APPROVED** [review]
LGTM~

File: pages/views/Performance.jsx [hunk 2]
```diff
@@ -120,7 +120,7 @@ const Performance = ({ portfolioApr, sharpeRatio, ROI, maxDrawdown }) => {
           margin: "32px 0",
         }}
       >
-        Historical Reward APRs
+        Historical Reward
```
[the remark this answers is not in the document]
Comment (fantzulin) [reply pages/views/Performance.jsx:123]: 那我覺得給用戶自行用按鈕切換會比較適合

**Pull request merged**